### ISSUE TITLE
Support for logging levels

### ISSUE THREAD

**Issue by jeremylockhart** [Reidmcc/rockfish#25]
Would it be possible to have options on the level of logging? Since all output is logged it is hard to parse back through and see the results. Ideally, I'm wanting to see transaction info if/when one is submitted and the result.

**Comment by Reidmcc** [comment]
@jeremylockhart This could be a good idea; when I go through the logs I often end up string searching for 'xdr' to find the transactions. Errors would of course remain logged.
